packages/kbn-grid-layout/grid/grid_panel.tsx

```diff
@@ -64,7 +65,8 @@ export const GridPanel = forwardRef<
           .pipe(skip(1)) // skip the first emit because the `initialStyles` will take care of it
           .subscribe(([activePanel, gridLayout, runtimeSettings]) => {
             const ref = gridLayoutStateManager.panelRefs.current[rowIndex][panelId];
-            const panel = gridLayout[rowIndex].panels[panelId];
+            const allPanels = gridLayout[rowIndex].panels;
```

Comment:
I don't think we need `allPanels` anymore - can we undo the unrelated changes to this subscription?

Reply:
done!